contracts/foundation/application.go

```diff
@@ -20,8 +21,20 @@ type Application interface {
 	DatabasePath(path string) string
 	// StoragePath get the path to the storage directory.
 	StoragePath(path string) string
+	// LangPath get the path to the language files.
+	LangPath(path string) string
 	// PublicPath get the path to the public directory.
 	PublicPath(path string) string
 	// Publishes register the given paths to be published by the "vendor:publish" command.
 	Publishes(packageName string, paths map[string]string, groups ...string)
+	// GetLocale get the current application locale.
+	GetLocale(ctx context.Context) string
+	// SetLocale set the current application locale.
+	SetLocale(ctx context.Context, locale string) context.Context
+	// SetLocaleByHttp set the current application/context locale by http request.
+	//SetLocaleByHttp(ctx http.Context, locale string)
```

Comment:
Q: Why remove this?

Reply:
I don't think we need this function we can just use `SetLocale` and pass http context.
```go
// ctx is http context
Lang(ctx).SetLocale(locale)
```